src/libraries/Microsoft.Extensions.Hosting/src/HostBuilder.cs

```diff
@@ -21,6 +21,9 @@ namespace Microsoft.Extensions.Hosting
     /// </summary>
     public partial class HostBuilder : IHostBuilder
     {
+        private const string _hostBuildingEventName = "HostBuilding";
```

Comment:
`const` strings shouldn't have `_` in their names.

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

12. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.